apps/condo/domains/ai/utils/serverSchema/removeSensitiveDataFromObj.js

```diff
@@ -0,0 +1,26 @@
+const { DeepRedact } = require('@hackylabs/deep-redact')
+
+function getPartialStringTest (pattern, replaceText) {
+    return {
+        pattern: pattern,
+        replacer: (value, pattern) => value.replaceAll(pattern, `${replaceText}`),
+    }
+}
+
+const deepRedact = new DeepRedact({
+    partialStringTests: [
+        getPartialStringTest(/(\+?\d{1,4}[\s-]?)?(\(?\d{2,4}\)?[\s-]?)?[\d\s-]{6,14}\d/g, '[phone]'),
+        getPartialStringTest(/\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b/g, '[email]'),
```

Comment:
It would be great to have a good name for these regular expressions.
If it weren't for the second argument, it would be difficult to understand what it is :)